source/reference/options.txt

```diff
@@ -76,6 +76,34 @@ General Options
 
    Returns the |mdb-shell| release number.
 
+.. _mongosh-stable-api-options:
+
+Stable API Options
+------------------
+
+.. option:: --apiVersion <version number>
+
+   Specifies the :ref:`apiVersion <api-version-desc>`. ``"1"`` is 
+   currently the only supported value.
+
+.. option:: --apiStrict
+
+   Specifies that the server will respond with :ref:`APIStrictError 
+   <api-strict-resp>` if your application uses a command or behavior 
+   outside of the :ref:`Stable API <stable-api>`.
+   
+   If you specify :option:`--apiStrict`, you must also specify
+   :option:`--apiVersion`.
+
+.. option:: --apiDeprecationErrors
+
+   Specifies that the server will respond with :ref:`APIDeprecationError
+   <api-deprecation-resp>` if your application uses a command or behavior 
+   that is deprecated in the specified :ref:`apiVersion 
```

Comment:
(linking)
This is the second instance of the apiVersion link. Fine to omit.

Reply:
Omitted.